docs/mscolab.rst

```diff
@@ -150,75 +150,22 @@ by `pg_dump <https://www.postgresql.org/docs/current/app-pgdump.html>`_ using a
-
-    mamba activate instance
-    cd ~/INSTANCE/config
-    export PYTHONPATH=`pwd`
-    cd ~/INSTANCE/wsgi
-    flask --app mscolab.py db init
-    flask --app mscolab.py db migrate -m "To version 9.0.0"
-    flask --app mscolab.py db upgrade
-
-The migration script builder does the base but the created script needs first allow nullable so that we afterwards set the default for the existing data.
-Use this as an example for your script ::
-
-    """To version 9.0.0
-
-    Revision ID: e62d08ce88a4
-    Revises: 27a026b0daec
-    Create Date: 2024-06-07 16:53:43.314338
-
-    """
-    from alembic import op
-    import sqlalchemy as sa
-
-
-    # revision identifiers, used by Alembic.
-    revision = 'e62d08ce88a4'
-    down_revision = '27a026b0daec'
-    branch_labels = None
-    depends_on = None
-
-
-    def upgrade():
-        # ### commands auto generated by Alembic - please adjust! ###
-        with op.batch_alter_table('users', schema=None) as batch_op:
-            batch_op.add_column(sa.Column('authentication_backend', sa.String(length=255), nullable=True))
-        op.execute('UPDATE users SET authentication_backend = \'local\' WHERE authentication_backend IS NULL')
-
-        with op.batch_alter_table('users', schema=None) as batch_op:
-            batch_op.alter_column('authentication_backend', existing_type=sa.String(length=255), nullable=False)
-            batch_op.drop_constraint('users_password_key', type_='unique')
-
-        # ### end Alembic commands ###
-
-
-    def downgrade():
-        # ### commands auto generated by Alembic - please adjust! ###
-        with op.batch_alter_table('users', schema=None) as batch_op:
-            batch_op.create_unique_constraint('users_password_key', ['password'])
-            batch_op.drop_column('authentication_backend')
-
-        # ### end Alembic commands ###
-
-
-The output looks like ::
-
-    ~/INSTANCE/wsgi$ flask --app mscolab.py db upgrade
-    INFO  [alembic.runtime.migration] Context impl SQLiteImpl.
-    INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
-    INFO  [alembic.runtime.migration] Running upgrade  -> e62d08ce88a, To version 9.0.0
-
+Database Migration from Version 8 or 9
+.................................
+
+From v10 onwards MSColab uses `Flask-Migrate <https://flask-migrate.readthedocs.io/en/latest/>` to automatically deal with database migrations.
+To upgrade from v8 or v9 a recreation of the database and subsequent copy of existing data is necessary.
+To do this follow these steps:
+
+#. **Make a backup of your existing database**
+#. Set ``SQLALCHEMY_DB_URI_TO_MIGRATE`` to your existing database
+#. Set ``SQLALCHEMY_DB_URI`` to a new database
+#. Start MSColab
+#. Check that everything was migrated successfully
+#. Unset ``SQLALCHEMY_DB_URI_TO_MIGRATE``
+
+If you want to keep using your old database name you can first move your existing database to a different location
```

Comment:
this is sqlite specific

Reply:
Only if you define "move" as "move on the filesystem". You can also move a PostgreSQL, or whatever else, database to a different location by renaming it. The point is that one can move the database to a different URI.

Rename might be the better verb for this sentence.